*** jshon/Makefile	2011-04-02 08:43:10.000000000 +1000
--- Makefile	2012-09-10 12:22:29.000000000 +1000
***************
*** 2,8 ****
  
  CC      ?=  gcc
  CFLAGS  += -std=c99 -Wall -pedantic -Wextra -Werror
! LDFLAGS += -ljansson
  
  SRC = jshon.c
  OBJ = ${SRC:.c=.o}
--- 2,8 ----
  
  CC      ?=  gcc
  CFLAGS  += -std=c99 -Wall -pedantic -Wextra -Werror
! LDFLAGS += -ljansson -L${JANSSON}/src/.libs
  
  SRC = jshon.c
  OBJ = ${SRC:.c=.o}
***************
*** 12,18 ****
  all: jshon
  
  .c.o:
! 	${CC} -c ${CFLAGS} $<
  
  ${OBJ}:
  jshon: ${OBJ}
--- 12,18 ----
  all: jshon
  
  .c.o:
! 	${CC} -c ${CFLAGS} -I ${JANSSON}/src $<
  
  ${OBJ}:
  jshon: ${OBJ}
